sources/packages/web/src/views/aest/institution/ProgramDetails.vue

```diff
@@ -1,29 +1,71 @@
 <template>
   <v-container>
-    <div class="mb-4">
-      <p class="muted-heading-text">
-        <a @click="goBack()">
-          <v-icon left> mdi-arrow-left </v-icon> Back all programs</a
-        >
-      </p>
-      <p class="heading-x-large">View program</p>
-    </div>
+    <header-navigator
+      title="Back all programs"
+      :routeLocation="{
+        name: AESTRoutesConst.INSTITUTION_PROGRAMS,
+        params: { institutionId: institutionId },
+      }"
+      subTitle="View program"
+    >
+      <template #buttons>
+        <div v-if="isPendingProgram">
+          <v-btn
+            outlined
+            :color="COLOR_BLUE"
```

Comment:
So far we have not tested the way of setting the theme colors of Vue application(we discussed about it) with primary and secondary colors with figma based colors. If we know for sure the theming is going to work(uncertain about alpha version), we can just add primary here and fix the theme across application. Otherwise we are using color codes now and will do a search and replace of :color="COLOR_BLUE" with color="primary" when Vuetify is ready.

Reply:
i think we can create a sep ticket for theming and color